asf_tools/README.md

```diff
@@ -45,6 +45,13 @@ To generate a composite of the cross-polarization images, navigate to the direct
 make_composite VH-composite */*VH.tif
 ```
 
+#### Usage Tip
+Because the imagery has been radiometrically terrain corrected (RTC), geometric and radiometric distortions have been removed from the files to be mosaicked.  One the strong points of LRW composite mosaics is that you combine both ascending and descending datatakes into a singe mosaic. In this manner no layover or shadow masks are required - what is shadowed on an ascending pass is visible in a descending pass and vice-versa.  Thus, not only is it possible to combine ascending and descending, but it is highly encouraged.  Using many datatakes from both the ascending and descending satellite passes will make the best mosaics possible.
```

Comment:
mosaicked --> composited
mosaics --> composites x2
mosaic --> composite

To be inline with how D. Small discusses these products and the rest of the documentation.

Reply:
Done.  Not even gonna let me sneak in a single mosaic...?  Geez.

Reply:
haha!